riptide-spring-boot-autoconfigure/src/main/java/org/zalando/riptide/autoconfigure/RiptideProperties.java

```diff
@@ -35,6 +36,7 @@
 @Setter
 @NoArgsConstructor
 @AllArgsConstructor
+@Component
```

Comment:
Interesting that you need to add this annotation as `@ConfigurationProperties` should normally make this available to Spring.

Reply:
Intellij showed an error saying `not registered via @enableconfigurationproperties or marked as spring component.`

The project compiled anyway, but I followed the advice here:
https://stackoverflow.com/a/57950415/459391

> This is expected as @ConfigurationProperties does not make a class a Spring Component. Mark the class with @Component and it should work. Note that a class can only be injected if it is a Component.

Reply:
Thanks for the insights. Yes, I remember that you have to add this annotation to your main class for example, to get this working. I guess, given that this is a library, it makes sense to follow that approach.